### PR TITLE
Add TiC-CLIP

### PR DESCRIPTION
**Pull request by merveenoyan**
cc @osanseviero @pcuenca as discussed
added it to have the model counts rather than having it as a library

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -422,6 +422,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/TensorSpeech/TensorFlowTTS",
 		snippets: snippets.tensorflowtts,
 	},
+	"tic-clip": {
+		prettyLabel: "tic-clip",
+		repoName: "tic-clip",
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts]:
```suggestion
		prettyLabel: "TiC-CLIP",
		repoName: "TiC-CLIP",
```

(nit) Taken from their repo

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -422,6 +422,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
...
+		repoUrl: "https://github.com/apple/ml-tic-clip",
+		filter: false,
+		countDownloads: { term: { path: "checkpoints/*.pt" } },
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:430]: This should be a wildcard as  https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts#L111

```suggestion
		countDownloads: { wildcard: { path: "checkpoints/*.pt" } },
```

File: packages/tasks/src/model-libraries.ts [hunk 3]
```diff
@@ -422,6 +422,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/TensorSpeech/TensorFlowTTS",
 		snippets: snippets.tensorflowtts,
 	},
+	"tic-clip": {
+		prettyLabel: "tic-clip",
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:426]: As in https://github.com/apple/ml-tic-clip

```suggestion
		prettyLabel: "TiC-CLIP",
```

File: packages/tasks/src/model-libraries.ts (hunk 1 above)
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:427]:
```suggestion
		repoName: "TiC-CLIP",
```

**Review by osanseviero — COMMENTED** [review]
Nice!

**Review by Wauplin — APPROVED** [review]
LGTM :)

To make it work, all models from https://huggingface.co/collections/apple/tic-clip-666097407ed2edff959276e0 must have `library_name: tic-clip` in their modelcard metadata. While you are at it, can you also explicitly set `pipeline_tag: zero-shot-image-classification` instead of having it as `tags`? (already works well with moon-landing but better to be explicit)

File: packages/tasks/src/model-libraries.ts (hunk 1 above)
Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts]: lol, too late

**Review by osanseviero — APPROVED** [review]
LGTM! Let's wait for one extra reviewer

**Comment by merveenoyan** [comment]
@Wauplin I can open PR to all once this is merged IMO (or should I do it before?)

**Comment by osanseviero** [comment]
It can be done in parallel - there's nothing blocking on either side. In general having proper metadata on the model cards, even if download stats are not enabled yet, would be good to have the correct library set up

**Review by julien-c — APPROVED** [review]
lgtm

**Pull request merged**